src/test/unittest/utils.py

```diff
@@ -46,3 +47,23 @@
 GiB = 2 ** 30
 TiB = 2 ** 40
 PiB = 2 ** 50
+
+
+def windows_only(tc):
+    """
+    Run test case (TEST[number] class) only on Windows.
+    Use it as a class decorator.
+    """
+    if sys.platform != 'win32':
+        tc.enabled = False
+    return tc
+
+
+def linux_only(tc):
+    """
+    Run test case (TEST[number] class) only on Linux.
+    Use it as a class decorator.
+    """
+    if sys.platform != 'linux':
+        tc.enabled = False
+    return tc
```

Comment:
What about FreeBSD?

I think we should also have exclude_[linux|windows|freebsd] variants of these decorators.

Reply:
Done.